pkg/endpoints/credentials.go

```diff
@@ -153,6 +156,20 @@ func (r Resource) createCredential(request *restful.Request, response *restful.R
 		return
 	}
 
+	// Patch service account
+	saName := ""
+	providedSaName := os.Getenv("PIPELINE_RUN_SERVICE_ACCOUNT")
+	if providedSaName != "" {
+		saName = providedSaName
+	} else {
+		saName = "default"
+	}
+	if !r.addSecretToSA(saName, cred.Name, requestNamespace) {
+		errorMessage := fmt.Sprintf("Error adding secret in service account: %s", saName)
```

Comment:
Should be lower case e for the error message string

Reply:
I fix it.